assets/js/index.js

```diff
@@ -0,0 +1,458 @@
+    text: "Which player has scored the most free-kicks? ",
+    options: ["David Beckham", "Ronaldo", "Messi", "Drogba"],
+    answer: "David Beckham",
+  },
+];
+// function to render banner
+const renderBanner = () => {
+  const section = document.createElement("section");
+  section.setAttribute(
+    "class",
+    " content-section animate__animated animate__bounceInDown"
+  );
+  section.setAttribute("id", "banner");
+  const h1 = document.createElement("h1");
+  h1.setAttribute("class", "content-section-title");
+  h1.textContent = "welcome to the hardest premier league quiz";
+  const h2 = document.createElement("h2");
+  h2.setAttribute("class", "content-section-info");
+  h2.textContent = "enter at your own risk";
+  const btnDiv = document.createElement("div");
+  btnDiv.setAttribute(
+    "class",
+    "btn-control animate__animated animate__bounceInDown"
+  );
+  const button = document.createElement("button");
+  button.setAttribute("class", "btn");
+  button.setAttribute("id", "start-btn");
+  button.textContent = "start";
+  btnDiv.append(button);
+  section.append(h1, h2, btnDiv);
+  main.append(section);
+  button.addEventListener("click", onStartButtonClick);
+};
+// function to restart
+const renderRestart = () => {
+  const section = document.createElement("section");
+  section.setAttribute("class", " content-section");
+  section.setAttribute("id", "restart-container");
+  const h2 = document.createElement("h2");
+  h2.setAttribute("class", "content-section-title");
+  h2.textContent = "wanna try again?";
+  const button = document.createElement("a");
+  button.setAttribute("class", "btn");
+  button.setAttribute("href", "./index.html");
+  button.textContent = "restart";
+  section.append(h2, button);
+  main.append(section);
+};
+//function to render results
+const renderResults = () => {
+  const tableArr = JSON.parse(localStorage.getItem("feedbackResults"));
+  //create section
+  const section = document.createElement("section");
+  section.setAttribute("class", "content-section");
+  section.setAttribute("id", "results-container");
+  // create h1
+  const title = document.createElement("h2");
+  title.setAttribute("class", "content-section-title");
+  title.textContent = "results";
+  //create scoreboard
+  const h2 = document.createElement("h2");
+  h2.setAttribute("class", "content-section-info");
+  h2.textContent = "your score was: " + scoreIndex + "/" + questions.length;
+  //create scoreboard
+  const timer = document.createElement("h2");
+  timer.setAttribute("class", "content-section-info");
+  timer.textContent = "time remaining: " + timeCountdown + " seconds";
+  // create div container for results
+  const div = document.createElement("div");
+  div.setAttribute("class", "results-section");
+  // create 2 uls with 8 li
+  const ul = document.createElement("ul");
+  ul.setAttribute(
+    "class",
+    "results-list animate__animated animate__bounceInDown"
+  );
+  const li1 = document.createElement("li");
```

Comment:
This is a lot of unnecessary repeated code - we're in javascript so think about how you can avoid repeating the same lines of code over and over again. We have different ways of iterating over arrays that will let us run the same functionality repeatedly, with much less code written.